gdcdictionary/schemas/virus_genome.yaml

```diff
@@ -87,11 +87,14 @@ properties:
     enum:
       - GenBank
       - NCBI
+
+  # Prop originated from Sars-cov2 project
   accession:
     description: >
       The accession id of the virus genome.
     type: string
 
+# Prop originated from COXRAY project
```

Comment:
There is no accession_date in the COXRAY project. I believe this is a GDC legacy prop.